### PR TITLE
instance.toString is not a function (close #561)

### LINKED ISSUE #561: instance.toString is not a function
```
Error
    at Object.eval (eval at evaluate (unknown source), <anonymous>:1:2)
    at Object.InjectedScript._evaluateOn (<anonymous>:904:55)
    at Object.InjectedScript._evaluateAndWrap (<anonymous>:837:34)
    at Object.InjectedScript.evaluateOnCallFrame (<anonymous>:963:21)
    at Object.HammerheadClient.define.exports.isStyleInstance (http://kirov-sv:1340/tcse6be4085-e27c-46d7-8310-fa770c1f18fa/hammerhead.js:25552:23)
    at Object.HammerheadClient.define.exports.init.elementPropertyAccessors.background.condition (http://kirov-sv:1340/tcse6be4085-e27c-46d7-8310-fa770c1f18fa/hammerhead.js:21606:33)
    at setProperty (http://kirov-sv:1340/tcse6be4085-e27c-46d7-8310-fa770c1f18fa/hammerhead.js:21775:52)
    at http://kirov-sv:1340/ace/ace.js?7929ba6d39aa4465=http%3A%7Clocalhost%3A3777%7C4%7Ct%7Cscript:10:86387
    at Array.forEach (native)
    at createKeywordMapper (http://kirov-sv:1340/ace/ace.js?7929ba6d39aa4465=http%3A%7Clocalhost%3A3777%7C4%7Ct%7Cscript:10:86270)
```

### PR DESCRIPTION
**Pull request by churkin**
\cc @VasilyStrelyaev @miherlosev

### CODE REVIEW

File: src/client/utils/style.js [hunk 1]
```diff
@@ -17,7 +17,7 @@ export function isStyle (instance) {
     if (instance instanceof nativeMethods.styleClass)
         return true;
 
-    if (instance && typeof instance === 'object' && instance.border !== void 0) {
+    if (instance && typeof instance === 'object' && instance.border !== void 0 && typeof instance.toString === 'function') {
```
Comment (inikulin) [inline src/client/utils/style.js]: Just use `String(instance)`

File: test/client/fixtures/utils/style-test.js [hunk 2]
```diff
@@ -231,3 +231,9 @@ test('getSelectElementSize', function () {
 
     document.body.removeChild(select);
 });
+
+module('regression');
+
+test('isStyle throws a error for the object without toString method (GH-561)', function () {
```
Comment (VasilyStrelyaev) [inline test/client/fixtures/utils/style-test.js]:
```
test('isStyle throws an error for objects without the toString method (GH-561)', function () {
```

**Comment by VasilyStrelyaev** [comment]
\r-

File: src/client/utils/style.js (hunk 1 above)
Comment (churkin) [inline src/client/utils/style.js]: `String({ border: 0, toString: null}) `

throws 

`Uncaught TypeError: Cannot convert object to primitive value`

Comment (inikulin) [inline src/client/utils/style.js]: ok

**Comment by miherlosev** [comment]
lgtm

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**